service/src/main/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperService.java

```diff
@@ -404,6 +405,15 @@ public void addResponseElementsToDirections(List<Element<DirectionResponse>> res
             .collect(groupingBy(directionElement -> directionElement.getValue().getAssignee()));
     }
 
+    /**
+     * Adds any {@link DirectionAssignee} not present to a map with an empty list of directions.
+     *
+     * @param map assignee, directions key value pairs.
+     */
+    public void addEmptyDirectionsForAssigneeNotInMap(Map<DirectionAssignee, List<Element<Direction>>> map) {
+        stream(DirectionAssignee.values()).forEach(assignee -> map.putIfAbsent(assignee, new ArrayList<>()));
```

Comment:
Was it not going to be `emptyList()` instead of `new ArrayList<>()`

Reply:
I get an issue in test where I am trying to add directions to an immutable list with empty list 😕

Reply:
Thought that might have been the issue